cohort-evaluator-hapi-fhir/src/main/java/com/ibm/cohort/cql/hapi/R4ParameterDefinitionWithDefaultToCohortParameterConverter.java

```diff
@@ -1,10 +1,10 @@
 /*
- * (C) Copyright IBM Corp. 2020, 2022
+ * (C) Copyright IBM Corp. 2022
```

Comment:
@rickjstevens Any thoughts on this?  I can go back and add the leading year manually, but I don't think IntelliJ's copyright templating system easily allows keeping the leading date.
I'm also not sure if any of this matters since the copyrights will be changing again later.

Reply:
Agree - would not advocate spending a lot of time on copyrights at this point in time.